src/test/java/AppTest.java

```diff
@@ -11,15 +11,17 @@
 import static org.junit.Assert.*;
 
 public class AppTest {
+
     private final ByteArrayOutputStream outcontent = new ByteArrayOutputStream();
     @Test public void testAppHasAGreeting() {
 
     }
     @Test public void tesGetRondomquote(){
-        Quote[] quotes = new Quote[]{new Quote("I love you God!","Hiwot")};
+        Quote[] quotes = new Quote[]{
```

Comment:
This test still doesn't test anything about the random quote getting method. It "works" by not doing any work.